public/assets/materialize/scrollFire-6699f19900c42c0c9d73784729c9238b16df695111c16a2ee6836e8275062d49.js

```diff
@@ -0,0 +1 @@
+!function(e){Materialize.scrollFire=function(e){var n=!1;window.addEventListener("scroll",function(){n=!0}),setInterval(function(){if(n){n=!1;for(var t=window.pageYOffset+window.innerHeight,i=0;i<e.length;i++){var o=e[i],r=o.selector,f=o.offset,l=o.callback,a=document.querySelector(r);if(null!==a){var c=a.getBoundingClientRect().top+window.pageYOffset;if(t>c+f&&o.done!==!0){var d=new Function(l);d(),o.done=!0}}}}},100)}}(jQuery);
```

Comment:
Line is too long.<br>Missing semicolon.<br>Confusing use of '!'.<br>The Function constructor is a form of eval.<br>Expected an assignment or function call and instead saw an expression.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>'Materialize' is not defined.<br>'e' is defined but never used.